cmd/chart.go

```diff
@@ -89,12 +89,16 @@ func moveChart(cmd *cobra.Command, args []string) error {
 		mover.WithRetries(retries), mover.WithLogger(cmd),
 	)
 	if err != nil {
-		if err == mover.ErrImageHintsMissing {
+		var loadingError *mover.ChartLoadingError
+		if errors.As(err, &loadingError) {
+			return loadingError
+		} else if err == mover.ErrImageHintsMissing {
 			return fmt.Errorf("image patterns file is required. Please try again with '--image-patterns <image patterns file>' or as part of the Helm chart at [chart]/%s file", mover.EmbeddedHintsFilename)
 		} else if err == mover.ErrOCIRewritesMissing {
 			return fmt.Errorf("at least one rewrite rule must be given. Please try again with --registry and/or --repo-prefix")
 		}
 
+		cmd.SilenceUsage = true
```

Comment:
I'd add a comment explaining why we need this silencing here.

I removed it once because I couldn't understand what was the purpose of having this override here.
Alternatively we could set silence to true in the top level and just call usage() when needed. That looks to me a more controlled way maybe.